data/data-pipeline/data_pipeline/content/config/csv.yml

```diff
@@ -14,6 +14,30 @@ fields:
   - score_name: State/Territory
     label: State/Territory
     format: string
+  - score_name: Percent Black or African Amer.
+    label: Percent Black or African American alone
+    format: float
+  - score_name: Percent Amer. Indian / Alaska Native
+    label: Percent Amer. Indian / Alaska Native
+    format: float
+  - score_name: Percent Asian
+    label: Percent Asian
+    format: float
+  - score_name: Percent Native Hawaiian or Pacific
+    label: Percent Native Hawaiian or Pacific
+    format: float
+  - score_name: Percent Two or more
+    label: Percent Two or more
+    format: float
+  - score_name: Percent White
+    label: Percent White
+    format: float
+  - score_name: Percent Hispanic or Latino
+    label: Percent Hispanic or Latino
+    format: float
+  - score_name: Percent Other
```

Comment:
nit: this won't be super clear in the download -- percent other race or ethnicity might be clearer?